speech/voices/azure.py

```diff
@@ -212,3 +210,34 @@ def text_to_audio_url(
     # mime_type = mimetype_for_audiofmt(audio_format)
     # data_uri = generate_data_uri(data, mime_type=mime_type)
     # return data_uri
+
+
+class Normalization(DefaultNormalization):
+    """
+    Normalization handler class,
+    specific to the Azure voice engine.
+    """
+
+    # Override some character pronunciations for
+    # normalization, custom for this speech engine
+    _CHAR_PRONUNCIATION = {
+        **DefaultNormalization._CHAR_PRONUNCIATION,
+        "b": "bjé",
+        "c": "sjé",
+        "d": "djé",
+        "ð": "eeð",
+        "e": "eeh",
+        "é": "jé",
+        "g": "gjé",
+        "i": "ii",
+        "j": "íoð",
+        "o": "úa",
```

Comment:
Really?!

Reply:
Yep :/ the Azure speech synthesis pronounces stuff weird, I found these to be closest to the actual pronunciation through trial and error.